torchtune/modules/transformer.py

```diff
@@ -10,8 +10,8 @@
 import torch.nn.functional as F
 from torch import nn
 from torchtune.modules import MultiHeadAttention
-
```

Comment:
PS: I forgot to update the typehint of the TransformerDecoder, saying that output can now be a callable. To avoid rerunning tests, this will come in a followin gemma PR